advanced_alchemy/extensions/litestar/cli.py

```diff
@@ -60,6 +60,8 @@ def show_database_revision(app: Litestar, verbose: bool) -> None:
 )
 def downgrade_database(app: Litestar, revision: str, sql: bool, tag: str | None, no_prompt: bool) -> None:
     """Downgrade the database to the latest revision."""
+    from advanced_alchemy.extensions.litestar.alembic import AlembicCommands
```

Comment:
I've also moved these here in an effort to reduce the amount of classes that are imported.